library/pod-security-policy/users/example.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
To demonstrate rejection/failure, we could either use an example that creates a pod or update the rego to look at deployment as well as the pod spec. The current example is a `Deployment` and the rego in the constraint template looks at the pod spec and containers. So this deployment will succeed while the pods will fail to create. This might create confusions for the user deploying this example.

Reply:
Good call @ritazh, I switched it to a pod example.